src/shaders.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+	'use strict';
+
+	var WAGNER = this.WAGNER || {};
+
+	WAGNER.shaders = {};
+
+	WAGNER.loadShader = function( id, cb ) {
```

Comment:
The api for this also needs to be resolved. I think it should be changed to be synchronous, and ShaderLoader be removed.